index.js

```diff
@@ -1,5 +1,16 @@
 'use strict';
 
+exports.getQueryString = function (maybeUrl) {
```

Comment:
Your call really, I'm not precious about it. But personally I always prefer the most explicit thing possible, hence `getQueryString`